pkg/framework/framework.go

```diff
@@ -227,20 +237,22 @@ func isOpenshiftCI() bool {
 // If not, returns '%current_directory%/_out'.
 func getCliOutputFilesPath() (string, error) {
 	if isOpenshiftCI() {
-		return filepath.Join(os.Getenv(ARTIFACT_DIR), "machine-api-e2e-suite"), nil
+		return filepath.Join(os.Getenv(artifactDir), "machine-api-e2e-suite"), nil
 	}
 
 	cwd, err := os.Getwd()
 	if err != nil {
-		return "", nil
+		return "", err
```

Comment:
Does this not cause a linter error, I thought all errors must be wrapped?

Reply:
It's because I temporarily disabled `wrapcheck` linter. There are too many errors to wrap, so it could be done in a separate PR.

Reply:
You could wrap this one as good practice ahead of the new linter being set up